Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.11 as build
+
+WORKDIR /
+COPY . /
+ENV CGO_ENABLED 0
+RUN go build -tags 'static' -ldflags '-extldflags "-static"' .
+
+FROM scratch
+LABEL maintainer="https://github.com/Ilyes512/boilr"
+
+COPY --from=build /boilr /
```

Comment:
Maybe also add these lines https://github.com/Ilyes512/kubia/blob/master/Dockerfile#L22-L24 It copies ssl certs (at this point it might not be needed cause we are not running a server), but adding `/etc/passwd` might be nice to add since `os/user` uses it as a fallback when building the app without cgo enabled.

I know that cobra (and maybe boilr as well - haven't checked it) use https://github.com/mitchellh/go-homedir at the moment, but that package can be replaced with `os/user` (the fallback was added in a later golang version).

Reply:
There is a build tag for static build with osuser. I'll find it.